Makefile

```diff
@@ -952,3 +952,10 @@ update-webassets:
 .PHONY: dronegen
 dronegen:
 	go run ./dronegen
+
+# backport auto-backports changes from one branch to other branches. 
+# The target will create a pull request on Github with the credentials 
+# stored in ~/.config/gh/hosts.yaml.
+.PHONY: backport
+backport:
+	cd ./assets/backport && go run . --to $(to) --pr $(pr) --owner $(owner) --repo $(repo)
```

Comment:
A couple of things:

1. Encapsulate the command in parenthesis. That way if the command fails, you won't be left in the `assets/backport` directory.
2. The Go flag package does not support `--` only supports `-` for flags. I actually tried this command and it doesn't work.
3. Is it required to set `owner` or `repo` here? It looks like they default to `gravitational` and `teleport` anyway.

```
# backport will automatically create backports for a given PR as long as you have the "gh" tool
# installed locally. To backport, type "make backport PR=1234 TO=branch/1,branch/2".
.PHONY: backport
backport:
    (cd ./assets/backport && go run main.go -pr=$(PR) -to=$(TO))
```

Reply:
1. Nice tip, I will do that!
2. The `flag` package supports both, https://pkg.go.dev/flag#hdr-Command_line_flag_syntax.
3. It's not required. I will remove those.